d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -1,11 +1,16 @@
 from pendulum import today
+import os
```

Comment:
These imports should all come above the `pendulum` import (since they belong to the standard library).

Reply:
(Having them in alphabetical order is also a good practice, in case you want to)